core/src/components/chip/chip.scss

```
@@ -37,6 +37,11 @@
   box-sizing: border-box;
 }
 
+:host(.chip-disabled) {
+  cursor: default;
+  opacity: .5;
```

```suggestion
  opacity: 0.4;
```

The MD spec has the opacity set to be 0.4 https://material.io/components/chips/